Shared/Samples/Identify graphics/IdentifyGraphicsView.swift

```diff
@@ -0,0 +1,110 @@
+    @State private var map = Map(basemapStyle: .arcGISTopographic)
+    
+    /// A graphics overlay containing a yellow polygon graphic.
+    @State private var graphicsOverlay = {
+        let graphicsOverlay = GraphicsOverlay()
+        
+        // Create a polygon from a list of points.
+        let polygon = Polygon(points: [
+            Point(x: -20e5, y: 20e5),
+            Point(x: 20e5, y: 20e5),
+            Point(x: 20e5, y: -20e5),
+            Point(x: -20e5, y: -20e5)
+        ])
+        
+        // Create a graphic of the polygon and add it to the overlay.
+        let polygonGraphic = Graphic(geometry: polygon)
+        graphicsOverlay.addGraphic(polygonGraphic)
+        
+        // Create a renderer using a simple fill symbol.
+        let polygonSymbol = SimpleFillSymbol(style: .solid, color: .yellow)
+        graphicsOverlay.renderer = SimpleRenderer(symbol: polygonSymbol)
+        
+        return graphicsOverlay
+    }()
+    
+    /// The tap location's screen point used in the identify operation.
+    @State private var tapScreenPoint: CGPoint?
+    
+    /// A Boolean value indicating whether to show identify result alert.
+    @State private var isShowingIdentifyResultAlert = false
+    
+    /// The message shown in the identify result alert.
+    @State private var identifyResultMessage = "" {
+        didSet { isShowingIdentifyResultAlert = identifyResultMessage.isNotEmpty }
+    }
+    
+    /// A Boolean value indicating whether to show an error alert.
+    @State private var isShowingErrorAlert = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error? {
+        didSet { isShowingErrorAlert = error != nil }
+    }
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(map: map, graphicsOverlays: [graphicsOverlay])
+                .onSingleTapGesture { screenPoint, _ in
+                    tapScreenPoint = screenPoint
+                }
+                .task(id: tapScreenPoint) {
+                    guard let tapScreenPoint else { return }
+                    
+                    do {
+                        // Identify the screen point on the graphics overlay using the proxy.
+                        // Use `identifyGraphicsOverlays` instead if you need to identify on
+                        // all the graphics overlay present in the map view.
+                        let identifyResult = try await mapViewProxy.identify(
+                            on: graphicsOverlay,
+                            screenPoint: tapScreenPoint,
+                            tolerance: 12
+                        )
+                        
+                        // Display an alert if any graphics were found at the screen point.
+                        if identifyResult.graphics.isNotEmpty {
+                            identifyResultMessage = "Tapped on \(identifyResult.graphics.count) graphic(s)."
```

Comment:
With the change to "graphic(s)" the screenshot doesn't match anymore. You could fix that with a new screenshot or a code change. I'm okay with either.